<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceConfigDialog</class>
 <widget class="QDialog" name="ServiceConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>240</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>191</y>
     <width>360</width>
     <height>40</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LeName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>服务名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>配置文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LeConfigFile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnSelectConfigFile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>服务程序</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LeProgramFile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnSelectProgramFile">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>工作目录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LeWorkDir">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>180</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnSelectWorkDir">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择目录</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LeName</tabstop>
  <tabstop>LeProgramFile</tabstop>
  <tabstop>BtnSelectProgramFile</tabstop>
  <tabstop>LeConfigFile</tabstop>
  <tabstop>BtnSelectConfigFile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
